### DISCUSSION TITLE
Vercel & Primsa - cannot find SSL .cert.pem

### DISCUSSION THREAD

**Discussion by ytsruh** [prisma/prisma#16358] [Q&A] [answered]
Hoping for some advice, not sure if this is more of a Prisma issue or a Vercel issue. 

I'm able to read an SSL certificate and connect to my database locally. However when I use the same relative path in a deployed envinroment to Vercel, the file can no longer be read & there a connection is refused.

My assumption is that the relative path to the prisma folder is incorrect after Vercel has done its build step. However I've no idea how to locate where the certiticate would be stored.

Does anybody have any advice or guidance?

**Comment by nurul3101** [comment, marked as answer]
Hello @ytsruh 👋 

Can you have a look at the solution mentioned in this [comment](https://github.com/prisma/prisma/issues/1673#issuecomment-647107891) on a related GitHub Issue, would this work in your case?

**Reply by ytsruh** [reply]
I did thank you,. Seems a little overkill to have to write to a temorary storage for a fairly simple problem.

To be fair, it absoultey does work. I would descro be it as more of a workaround than a solution

**Reply by nurul3101** [reply]
Yes, that's true. It definitely is a workaround and not an ideal solution. But glad to hear that it worked for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
